bats/tests/helpers/vm.bash

```diff
@@ -320,9 +325,45 @@ docker_context_exists() {
     assert_output "$RD_DOCKER_CONTEXT"
 }
 
+# Check if the VM is using systemd (instead of OpenRC).
+using_systemd() {
+    [[ -n ${RD_USING_SYSTEMD:-} ]] || load_var RD_USING_SYSTEMD || true
+    if [[ -z ${RD_USING_SYSTEMD:-} ]]; then
+        # `systemctl whoami` contacts the systemd init to check things, so if
+        # it succeeds we're using systemd.  On alpine-based systems, the
+        # `systemctl` command would be missing so this still applies.
+        if rdctl shell /usr/bin/systemctl whoami &>/dev/null; then
+            RD_USING_SYSTEMD=true
+        else
+            RD_USING_SYSTEMD=false
+        fi
+        save_var RD_USING_SYSTEMD
+    fi
+    "${RD_USING_SYSTEMD}"
```

Comment:
`using_systemd` is used as a predicate, so needs to set a return code, not echo true/false.

```suggestion
    is_true "${RD_USING_SYSTEMD}"
```